adaptive-education-ai/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+node_modules
```

Comment:
We should probably exclude the .vscode folder as well. It looks like the file contents are fairly well abstracted, but unless they need notable manual configuration, vscode will generate those for anyone who doesn't have them and it would be better not to have IDE specific stuff in the repo.